microservices-connector/README.md

```diff
@@ -36,83 +36,64 @@ There are `2` components in this repo:
 - 1. `manager`: controller manager to handle GMC CRD
 - 2. `router`: route the traffic among the microservices defined in GMC
 
-#### How to build these binaries?
+### GMC build
+
+#### Binaries building
 
 ```sh
 make build
 ```
 
-#### How to build docker images for these 2 components?
+#### Docker images building
 
 ```sh
 make docker.build
 ```
 
-#### How to delete these components' binaries?
+#### Binaries deleting
 
 ```sh
 make clean
 ```
 
-### To Deploy on the cluster
+### GMC Deployment on K8s cluster
 
-**Build and push your image to the location specified by `CTR_IMG`:**
+**GMC Images NOTE:** This image ought to be published in [dockhub](https://hub.docker.com/u/opea), including [gmcmanager](https://hub.docker.com/r/opea/gmcmanager) and [gmcrouter](https://hub.docker.com/r/opea/gmcrouter). Make sure you have the proper permission to the registry and use the latest images.
 
-```sh
-make docker.build docker.push CTR_IMG=<some-registry>/gmcmanager:tag
-```
+There are 2 methods for deploying GMC on K8s cluster:
 
-**NOTE:** This image ought to be published in the personal registry you specified.
-And it is required to have access to pull the image from the working environment.
-Make sure you have the proper permission to the registry if the above commands don’t work.
+- Deploy via native kubectl
+- Deploy via helm chart
 
-**Install GMC CRD**
+#### Deploy using native kubectl
 
-```sh
-kubectl apply -f config/crd/bases/gmc.opea.io_gmconnectors.yaml
-```
+There are 3 steps for deploying GMC on K8s cluster as below:
 
-**Copy GMC router manifest**
+- Install GMC CRD
+- Prepare GenAI Components and GMC Router manifests
+- Deploy GMC Manager
 
-```sh
-cp $(pwd)/config/gmcrouter/gmc-router.yaml -p $(pwd)/config/manifests/
-```
+**Deploy GMC NOTE:**
 
-**NOTE:**
-before apply the manifests, please replace your own huggingface tokens in the manifests
+- Before installting the manifests, please replace your own huggingface tokens
+- Please set the path to the manifests if you have pre-defined directory to save the models on you cluster hosts
+- The `SYSTEM_NAMESPACE` should keep the same with the namespace defined in `gmc-manager.yaml` and `gmc-manager-rbac.yaml`
+- The configmap name `gmcyaml` is defined in gmcmanager deployment Spec. Please modify accordingly if you want
+  use a different name for the configmap
 
```

Comment:
Please add an paragraph of describing `MOUNT_DIR`, what does it mean? which kind of steps the users need to do, e.g. make sure the MOUNT_DIR is ready on every node of the k8s nodes)